ctapipe/image/tests/test_sliding_window_correction.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+import astropy.units as u
+from numpy.testing import assert_allclose
+
+from ctapipe.image.extractor import SlidingWindowMaxSum
+from ctapipe.image.toymodel import WaveformModel
+from ctapipe.instrument import SubarrayDescription, TelescopeDescription
+
+from ctapipe.utils import datasets
+
+datasets.DEFAULT_URL = "http://cccta-dataserver.in2p3.fr/data/ctapipe-extra/v0.3.2/"
```

Comment:
Shouldn't really set this globally, since it affects all other tests (and definitely should not be committed when we merge the PR).   Perhaps for now, you might want to use the `monkeypatch` test fixture instead, something like:

```py3
def test_xxx(monkeypatch):
    monkeypatch.setattr(datasets, "DEFAULT_URL", "http://cccta-dataserver.in2p3.fr/data/ctapipe-extra/v0.3.2/")
   # the rest of the test

Reply:
hmm, I put this test explicitely into a separate file to avoid affecting other files with a global variable, because I thought that each file is run independently.
either way, now modified as suggested by you